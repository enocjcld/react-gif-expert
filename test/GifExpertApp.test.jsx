import { render } from "@testing-library/react";
import { GifExpertApp } from "../src/GifExpertApp";


describe('Pruebas del GifExpertApp', () => {

    test('debe de hacer match con el snaptshop', () => {

        const { container } = render( <GifExpertApp /> );
        expect( container ).toMatchSnapshot();
    });

    test('debe de comprobar que el onAddCategory reciba la categoria', () => {

        
    });

});