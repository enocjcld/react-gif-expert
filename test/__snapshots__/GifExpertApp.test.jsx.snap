// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas del GifExpertApp debe de hacer match con el snaptshop 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="Buscar gifs"
      type="text"
      value=""
    />
  </form>
  <h3>
    Dragon ball
  </h3>
  <h2>
    Cargando...
  </h2>
  <div
    class="card-grid"
  />
</div>
`;
